charts/ndm/values.yaml

```diff
@@ -0,0 +1,99 @@
+# Default values for ndm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+release:
+  version: "0.8.0"
+ndm:
+  componentName: ndm
+  enabled: true
+  image:
+    # Make sure that registry name end with a '/'.
+    # For example : quay.io/ is a correct value here and quay.io is incorrect
+    registry:
+    repository: openebs/node-disk-manager
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: 0.8.0
+  sparse:
+    path: "/var/openebs/sparse"
+    size: "10737418240"
+    count: "0"
+  updateStrategy:
+    type: RollingUpdate
+  annotations: {}
+  podAnnotations: {}
+  ## Labels to be added to ndm daemonset pods
+  podLabels:
+    name: openebs-ndm
+  nodeSelector: {}
+  tolerations: []
+  filters:
+    enableOsDiskExcludeFilter: true
+    enableVendorFilter: true
+    excludeVendors: "CLOUDBYT,OpenEBS"
+    enablePathFilter: true
+    includePaths: ""
+    excludePaths: "loop,fd0,sr0,/dev/ram,/dev/dm-,/dev/md,/dev/rbd"
+  probes:
+    enableSeachest: true
+    enableUdevProbe: true
+    enableSmartProbe: true
+  healthCheck:
+    initialDelaySeconds: 30
+    periodSeconds: 60
+
+ndmOperator:
+  name: openebs-ndm-operator
+  enabled: true
+  image:
+    registry:
+    repository: openebs/node-disk-operator
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: 0.8.0
+  annotations: {}
+  podAnnotations: {}
+  nodeSelector: {}
+  tolerations: []
+  healthCheck:
+    initialDelaySeconds: 30
+    periodSeconds: 60
+  readinessCheck:
+    initialDelaySeconds: 4
+    periodSeconds: 10
+    failureThreshold: 1
+  replicas: 1
+  upgradeStrategy: Recreate
+
+helperPod:
+  image:
+    registry: ""
+    repository: openebs/linux-utils
+    pullPolicy: IfNotPresent
+    # Overrides the image tag whose default is the chart appVersion.
+    tag: 2.0.0
+
+crd:
+  enableInstall: true
+
+featureGates:
+  enabled: true
+  GPTBasedUUID:
+    enabled: true
+    featureGateFlag: "GPTBasedUUID"
+  APIService:
+    enabled: true
+    featureGateFlag: "APIService"
+    address: "0.0.0.0:9115"
```

Comment:
```suggestion
    api-service-address: 0.0.0.0:9115
```

Reply:
Following is the output even it is quotes. Did a dry run using ( helm install --debug --dry-run )
```
      - name: openebs-ndm
        image: "openebs/node-disk-manager:0.8.0"
        args:
          - -v=4
          - --feature-gates=GPTBasedUUID
          - --feature-gates=APIService
          - --api-service-address=0.0.0.0:9115
        imagePullPolicy: IfNotPresent
```